extensions/elasticsearch-java-client/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,14 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Elasticsearch Java Client"
```

Comment:
I didn't notice this before, but I think documentation is missing, do you think you could add a page?

It doesn't need to be extensive, even just stating what the extension is, what the entry point is (the client class), and including the configuration reference would be a start.

Reply:
I planned to add a documentation and a quickstart later on but as this PR will be on ice for some time I'll do it in the same PR. 

The documentation will be on the already existing Elasticsearch guide and I'll update the section about the high level client to explain that the java one is prefered.